airbyte-integrations/connectors/source-slack/source_slack/spec.json

```diff
@@ -0,0 +1,144 @@
+{
+  "documentationUrl": "https://docs.airbyte.com/integrations/sources/slack",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Slack Spec",
+    "type": "object",
+    "required": ["start_date", "lookback_window", "join_channels"],
+    "additionalProperties": true,
+    "properties": {
+      "start_date": {
+        "type": "string",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
+        "description": "UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated.",
+        "examples": ["2017-01-25T00:00:00Z"],
+        "title": "Start Date"
```

Comment:
Let's add `format: date-time` to this field so we show a date-time picker in our UI